lib/google_search_data_viewer_web/templates/keyword/show.html.heex

```diff
@@ -0,0 +1,69 @@
+<section class="flex flex-col items-center justify-center gap-10">
+  <h1 class="text-2xl text-center"><%= @keyword_upload.name %></h1>
+
+  <label for="html-modal" class="btn modal-button">Show HTML</label>
+  <input type="checkbox" id="html-modal" class="modal-toggle" />
+  <label for="html-modal" class="modal cursor-pointer">
+    <label class="modal-box w-11/12 max-w-5xl" for="">
+      <%= @keyword_upload.html %>
+    </label>
+  </label>
+
+  <div class="card w-96 bg-neutral text-neutral-content">
+    <div class="card-body items-center text-center">
+      <h2 class="card-title">Summary</h2>
+      <p>Total links: <%= total_urls(@search_results) %> </p>
+      <p>Total Top Adwords: <%= total_top_adword_urls(@search_results) %> </p>
+      <p>Total Adwords: <%= total_adword_urls(@search_results) %> </p>
+      <p>Total Non Adwords: <%= total_non_adword_urls(@search_results) %> </p>
+    </div>
+  </div>
+
+  <div class="collapse collapse-arrow border border-base-300 bg-base-100 rounded-box w-11/12 max-w-6xl bg-primary">
+    <input type="checkbox" /> 
```

Comment:
Why do we need a checkbox here? 🤔

Reply:
I am guessing the `checked` status / event is used for displaying / hiding the modal with DaisyUI :)